tests/test_collapse.py

```diff
@@ -76,13 +74,51 @@ def test_reconcile_prefix(self):
         """Test curie reconciliation is performing as expected."""
         msdf = parse_sssom_table(data_dir / "basic3.tsv")
 
-        with open(prefix_recon_yaml) as pref_rec:
-            prefix_reconciliation = yaml.safe_load(pref_rec)
+        self.assertEqual(
+            {
+                "a": "http://example.org/a/",
+                "b": "http://example.org/b/",
+                "c": "http://example.org/c/",
+                "d": "http://example.org/d/",
+                "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+                "owl": "http://www.w3.org/2002/07/owl#",
+                "orcid": "https://orcid.org/",
+                "semapv": "https://w3id.org/semapv/vocab/",
+                "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+                "skos": "http://www.w3.org/2004/02/skos/core#",
+                "sssom": "https://w3id.org/sssom/",
+            },
+            msdf.prefix_map,
+        )
+        prefix_reconciliation = {
+            "prefix_synonyms": {
+                "a": "c",
+                "b": "bravo",
+                "r": "rdfs",
+                "o": "owl",
+            },
+            "prefix_expansion_reconciliation": {
```

Comment:
Rename `expansion_reconciliation` to `rewiring` to maintain consistent terminology?

Reply:
What about this?